gcp/table_gcp_projects_organization_policy.go

```diff
@@ -0,0 +1,183 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpProjectsOrganizationPolicy(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_projects_organization_policy",
+		Description: "GCP Projects Organization Policy",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getGcpProjectsOrganizationPolicy,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listGcpProjectsOrganizationPolicies,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The name of the Constraint the Policy is configuring.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Constraint").Transform(lastPathElement),
+			},
+			{
+				Name:        "update_time",
+				Description: "The time stamp the Policy was previously updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "version",
+				Description: "Version of the Policy. Default version is 0.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "etag",
+				Description: "An opaque tag indicating the current version of the Policy, used for concurrency control.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "list_policy",
+				Description: "List of values either allowed or disallowed.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "boolean_policy",
+				Description: "For boolean Constraints, whether to enforce the Constraint or not.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "restore_default",
+				Description: "Restores the default behavior of the constraint; independent of Constraint type.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Constraint"),
```

Comment:
Can we use `Transform:   transform.FromField("Constraint").Transform(lastPathElement),` for `title`?